tests/test_infra_status.py

```diff
@@ -107,13 +87,32 @@
             ),
         ),
         (
-            [],
+            [OUTAGE_FULL, OUTAGE_NO_ENDDATE, OUTAGE_NO_TICKET, OUTAGE_NO_ENDDATE_NO_TICKET],
             [],
             (
                 "I checked Fedora Status (http://status.example.com) and there are "
-                "**no planned or ongoing outages on Fedora Infrastructure.**"
+                "**4 ongoing** and **0 planned** outages on Fedora Infrastructure.\n\n"
+                "##### Ongoing\n"
+                "● **thetitle (https://i.test/1)**\n"
+                "   Started at: 2023-08-06T1:00:00+0000\n"
+                "   Estimated to end: 2023-08-06T2:00:00+0000\n"
```

Comment:
Will this test pass now that it should include relative times?

A library I've used in the past is https://pypi.org/project/freezegun/ to set a stable time to make it easier to write tests that do time calculations off the current time, in case that's why this doesn't include relative time changes.